io/eolearn/io/processing_api.py

```diff
@@ -164,6 +164,7 @@ def _add_meta_info(self, eopatch):
 ProcApiType = collections.namedtuple('ProcApiType', 'id unit sample_type np_dtype feature_type')
 
 
+# pylint: disable=R0913
```

Comment:
Could you add this into `__init__` method right after docs strings? If you leave it here outside I think it is applied to all functions and methods in this module.

I also recommend changing to `# pylint: disable=too-many-arguments`. This way it will be more human-readable than `R0913`.